draft/src/api.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Debug)]
```

Comment:
Combine derives into one line using commas. Try running clippy, the linter. Might just be "cargo clippy".

Reply:
Done, clippy didn't actually complain about this though.